WRW/models.py

```diff
@@ -0,0 +1,70 @@
+import settings as constant
+from random import choice
+import exceptions as exception
+
+
+class Enemy:
+    def __init__(self, level=1):
+        self.level = level
+        self.health_points = level
+
+    @classmethod
+    def level_up(cls, enemy):
+        return cls(enemy.level + 1)
+
+    def decrease_health(self):
+        self.health_points -= 1
+        if self.health_points < 1:
+            raise exception.EnemyDownError(self.level)
+
+    def select(self):
+        return choice(constant.CHOICES)
+    
+
+class Player:
+    def __init__(self, name):
+        self.name = name
+        self.health_poitns = constant.PLAYER_HEALTH
+        self.score = constant.STARTING_SCORE
+    
+    def decrease_health(self):
+        self.health_poitns -= 1
+        if self.health_poitns < 1:
+            raise exception.GameOverError(self)
+    
+    def select(self):
+        while True:
+            try:
+                choice = int(input("Warrior-1, Robber-2, Wizard-3: "))
+                if 1 > choice or choice > 3:
```

Comment:
Пам'ятаєш, я якось писав про інверсію умов?

```suggestion
                if 0 < choice <= 3:
```

Reply:
> Пам'ятаєш, я якось писав про інверсію умов?

Був в мене такий варіант) Але чомусь я ліпив not і це мені тоді не сподобалось)
if  not 0 < choice <= 3:
    print("Enter 1 or 2 or 3")